tutorial/hero-image-placeholder.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Placeholder generation for the hero image
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Harshith](https://www.linkedin.com/in/harshith-raj-092ba4176)_
+
+An image placeholder is a background color over the space, on which an actual image will be loaded.
+
+How Quintype handles image placeholder?
+
+- Wrap `ResponsiveImage` component with `figure` tag.
+- Based on aspect ratio provide required `padding-top` to the `figure` tag using styleName/className.
```

Comment:
Cannot understand this sentence. Please change it.

Reply:
done

Reply:
Change to this "Based on aspect ratio, you need to provide the required `padding-top` to the `figure` tag using styleName/className."

Reply:
here we are mentioning how quintype is handling, not how they need to do